go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/OctopusDeploy/cli
 
 go 1.19
 
+replace (
+	github.com/OctopusDeploy/go-octopusdeploy/v2 => ../go-octopusdeploy
+)
+
```

Comment:
We should remove this

Reply:
this is waiting on https://github.com/OctopusDeploy/go-octopusdeploy/pull/161 to be merged

Reply:
and then it will be removed